src/spatio_temporal_voxel_layer.cpp

```diff
@@ -62,15 +63,15 @@ void SpatioTemporalVoxelLayer::onInitialize(void)
 
   // initialize parameters, grid, and sub/pubs
   ros::NodeHandle nh("~/" + name_), g_nh, prefix_nh;
+  _nh = &nh;
```

Comment:
This is unused, can you remove _nh completely, I'm not sure why it was added in the other PR